src/style.css

```diff
@@ -1 +1 @@
pse{white-space:nowrap;overflow:hidden;text-overflow:ellipsis}.vl-u-text--uppercase{text-transform:uppercase}.vl-u-text--lowercase{text-transform:lowercase}.vl-u-text--capitalize::first-letter{text-transform:uppercase}.vl-u-text--success{color:#3ca854}.vl-u-text--warning{color:#ffc515}.vl-u-text--error{color:#db3434}.vl-u-text--bold{font-weight:500}.vl-u-text--italic{font-style:italic}.vl-u-text--small{font-size:1.4rem!important}.vl-u-text--xsmall{font-size:1.2rem!important}.vl-u-text--xxsmall{font-size:1rem!important}.vl-u-text--marked,mark{background-color:transparent;box-shadow:inset 0 -10px 0 0 rgba(255,197,21,.3)}@media screen and (min-width:1023px){.vl-u-visible--l{display:block!important}}@media screen and (max-width:1023px){.vl-u-visible--m{display:block!important}}@media screen and (max-width:767px){.vl-u-visible--s{display:block!important}}@media screen and (max-width:500px){.vl-u-visible--xs{display:block!important}}.vl-u-hidden{display:none}@media screen and (min-width:1023px){.vl-u-hidden--l{display:none!important}}@media screen and (max-width:1023px){.vl-u-hidden--m{display:none!important}}@media screen and (max-width:767px){.vl-u-hidden--s{display:none!important}}@media screen and (max-width:500px){.vl-u-hidden--xs{display:none!important}}.vl-u-whitespace{white-space:normal}.vl-u-whitespace--nowrap{white-space:nowrap}.vl-u-whitespace--pre{white-space:pre}.vl-u-whitespace--pre-line{white-space:pre-line}.vl-u-whitespace--pre-wrap{white-space:prewrap}.vl-u-whitespace--unset{white-space:unset}.vl-u-whitespace--break-spaces{white-space:break-spaces}:root{--vl-theme-primary-color:#ffe615;--vl-theme-primary-color-60:#fff073;--vl-theme-primary-color-70:#ffee5b;--vl-theme-primary-color-rgba-30:rgba(255, 230, 21, 0.3);--vl-theme-fg-color:#333332;--vl-theme-fg-color-60:#858584;--vl-theme-fg-color-70:#707070}:host{display:none;position:relative;--vl-map--margin-top:0px;height:calc(var(--vl-map-height,500px) - var(--vl-map--margin-top))}#map{position:relative;height:calc(var(--vl-map-height,500px) - var(--vl-map--margin-top));width:100%;border:1px solid #cbd2da;box-sizing:border-box;overflow:hidden;background:#fff}#map .ol-rotate,#map .ol-zoom{margin-top:var(--vl-map--margin-top)!important}#map .vl-map-search__overlaycontainer{display:initial!important;width:100%}#map .vl-map-search__overlaycontainer vl-map-search{--vl-search-width:100%;position:absolute;margin:10px;box-sizing:border-box}@media screen and (max-width:500px){#map .vl-map-search__overlaycontainer vl-map-search{width:100%}}@media screen and (min-width:500px) and (max-width:767px){#map .vl-map-search__overlaycontainer vl-map-search{width:100%}}@media screen and (min-width:767px) and (max-width:1023px){#map .vl-map-search__overlaycontainer vl-map-search{width:50%}}@media screen and (min-width:1023px) and (max-width:1600px){#map .vl-map-search__overlaycontainer vl-map-search{width:50%}}@media screen and (min-width:1600px){#map .vl-map-search__overlaycontainer vl-map-search{width:50%}}
```

Comment:
ter info, waar halen we deze .css. Komt dit van web universum?

Reply:
Ja inderdaad, via een script wordt dat er op gezet